gradle/defaults.gradle

```diff
@@ -77,4 +68,13 @@ allprojects {
       }
     }
   }
+
+  // Repositories to fetch dependencies from.
+  repositories {
+    mavenCentral()
+    maven {
+      name "ApacheSnapshots"
+      url propertyOrDefault('apache-snapshot-repo', 'https://repository.apache.org/content/repositories/snapshots/')
```

Comment:
Does it need to have "Apache" in the name?  With the URL configurable, associating with Apache seems wrong/misleading.